### PR TITLE
Shifted Vite configs to different trouble shooting page

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

File: docs/troubleshooting/nodejs-polyfill-issues.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: NodeJS Polyfills issue
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Webpack 5 Issues | Documentation - Web3Auth"
```
Comment (shahbaz17) [inline docs/troubleshooting/nodejs-polyfill-issues.mdx:5]: Update the description please

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 2]
```diff
@@ -191,6 +191,7 @@ module.exports = {
       Troubleshooting: [
         "troubleshooting/error-429",
         "troubleshooting/webpack-issues",
+        "troubleshooting/nodejs-polyfill-issues",
         "troubleshooting/jwt-errors",
```
Comment (yashovardhan) [inline sidebars.js:195]: rename the page as well

**Comment by yashovardhan** [comment]
Hey @Saviour1001 please update this PR as discussed

**Comment by yashovardhan** [comment]
@Saviour1001 please update this

File: docs/troubleshooting/bundler-polyfill-issues.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      ],
+    },
+  },
+});
+```
+
+Even after adding this you might face issues like an empty modal popping up. The empty modal just pops up locally, it happens due to caching issues.
+You can remove the `React.StrictMode` from your `index.js` file to fix this issue.
+
+The empty modal doesn't appear in production builds.
```
Comment (yashovardhan) [inline docs/troubleshooting/bundler-polyfill-issues.mdx:90]: This should be in :::caution block

**Review by yashovardhan — CHANGES_REQUESTED** [review]

**Review by yashovardhan — APPROVED** [review]
LGTM

**Pull request merged**